### PR TITLE
support deployed models without version tag

### PR DESCRIPTION
**Pull request by BenFuernrohr**

### CODE REVIEW

File: src/main/java/info/novatec/camunda/migrator/ProcessVersion.java [hunk 1]
```diff
@@ -11,6 +11,9 @@
     private final int patchVersion;
 
     public static ProcessVersion fromString(String versionString) {
+    	if (versionString == null) {
```
Comment (tobiasschaefer) [inline src/main/java/info/novatec/camunda/migrator/ProcessVersion.java:14]: please check indentation

File: src/test/java/info/novatec/camunda/migrator/ProcessInstanceMigratorTest.java [hunk 2]
```diff
@@ -310,6 +309,75 @@ public void processInstanceMigrator_should_not_migrate_to_higher_major_version()
...
+
+        assertThat(getCurrentTasks(PROCESS_DEFINITION_KEY))
+            .numberOfTasksIs(2)
+            .allTasksHaveDefinitionId(newestProcessDefinition.getId())
+            .allTasksHaveFormkey(null);
+    }
+    
+    @Test
+    public void processInstanceMigrator_should_not_fail_if_only_process_models_without_version_tag_exist() {
+    	cleanUp();
```
Comment (tobiasschaefer) [inline src/test/java/info/novatec/camunda/migrator/ProcessInstanceMigratorTest.java:342]: wouldn't it be nicer to generally have the "cleanup" called via `@Before`?

File: src/main/java/info/novatec/camunda/migrator/ProcessVersion.java [hunk 3]
```diff
@@ -11,6 +11,9 @@
     private final int patchVersion;
 
     public static ProcessVersion fromString(String versionString) {
```
Comment (tobiasschaefer) [inline src/main/java/info/novatec/camunda/migrator/ProcessVersion.java:13]: wouldn't it be nicer to return an `Optional<ProcessVersion>` here? Seems better to me than mapping null to 0.0.0.

**Review by tobiasschaefer — COMMENTED** [review]
Hi @BenFuernrohr , I added some comments

File: src/main/java/info/novatec/camunda/migrator/ProcessVersion.java (hunk 1 above)
Comment (BenFuernrohr) [reply src/main/java/info/novatec/camunda/migrator/ProcessVersion.java:14]: Fixed indentation.

File: src/test/java/info/novatec/camunda/migrator/ProcessInstanceMigratorTest.java (hunk 2 above)
Comment (BenFuernrohr) [reply src/test/java/info/novatec/camunda/migrator/ProcessInstanceMigratorTest.java:342]: The whole construct was a little dirty. I refactored the tests somewhat to make the setup more consistent. Cleanup is and will be called via @After.

File: src/main/java/info/novatec/camunda/migrator/ProcessVersion.java (hunk 3 above)
Comment (BenFuernrohr) [reply src/main/java/info/novatec/camunda/migrator/ProcessVersion.java]: Agreed! Refactored the whole thing to use an Optional instead of Version 0.0.0.

**Pull request merged**